hardware/Bender.yml

```diff
@@ -41,7 +41,7 @@ dependencies:
   pulp_cluster: { git: "git@github.com:AlSaqr-platform/pulp_cluster.git", rev: "d47f3d0d207a5f22f301706f825dbf960f97f56b" } # branch: master
   can_bus: { git: "git@github.com:alsaqr-platform/can_bus.git", rev: "0ec0bf8b7dab6d5e4b3f7ec58338a8efee066379" }
   common_pads: { git: "git@github.com:AlSaqr-platform/common_pads.git", version: 0.0.4 }
-  opentitan: { git: "https://github.com/AlSaqr-platform/opentitan.git", rev: "122cd13195437362a533e9dd7863a33554f81f83" }
+  opentitan: { git: "https://github.com/AlSaqr-platform/opentitan.git", rev: "de8a5f348467af057677d2928894e15bf74dd7af" }
```

Comment:
Why this commit is changed? Is this the opentitan implementation + the DMA?

Reply:
we needed to change the axi user width